doc-content/enterprise-only/installation/clustering-dev-con.adoc

```diff
@@ -20,10 +20,8 @@ ifdef::PAM[]
  to test those rules and processes.
 endif::[]
 
-To create a {PRODUCT} clustered development environment, you download and extract {EAP} {EAP_VERSION} and {PRODUCT} {PRODUCT_VERSION}. Then, you configure {EAP} {EAP_VERSION} with Elasticsearch and AMQ Broker, a Java messaging server (JMS) broker. Finally, you start the cluster and install {CENTRAL} on each cluster node.
+To create a {PRODUCT} clustered development environment, you download and extract {EAP} {EAP_VERSION} and {PRODUCT} {PRODUCT_VERSION}. Then, you configure {EAP} {EAP_VERSION} with {DATAGRID} {DATAGRID_VERSION_LONG} and AMQ Broker, a Java messaging server (JMS) broker. Finally, you start the cluster and install {CENTRAL} on each cluster node.
```

Comment:
"you" is not required here. We can just say "To create a {PRODUCT} clustered development environment, download and extract {EAP} ...."